### ISSUE TITLE
github.actions cannot handle non-ASCII characters in AUTHORS

### ISSUE THREAD

**Issue by jkeenan** [Perl/perl5#17776]
Our github.actions set-ups appear to be unable to handle certain non-ASCII characters in names in AUTHORS.  See, for [example](https://pipelines.actions.githubusercontent.com/F8yobFedDMKReLD2lUI49NJpt8TNtEKo3DptPtnjfqw2qokl1r/_apis/pipelines/1/runs/8973/signedlogcontent/3?urlExpires=2020-05-10T13%3A20%3A50.0870196Z&urlSigningMethod=HMACV1&urlSignature=8YBuJfPm%2Flqc8LVnCQhD7S82rIinLmZtvfQ7UrU6qo8%3D):
```
2020-05-07T19:27:57.9885679Z t/porting/args_assert .......................................... ok
2020-05-07T19:27:58.0828100Z HÃ¥kon HÃ¦gland  <hakon.hagland@gmail.com> not found in AUTHORS
2020-05-07T19:27:58.0884082Z t/porting/authors .............................................. FAILED at test 1
2020-05-07T19:27:58.1052223Z t/porting/bench ................................................ skipped
```
However, I successfully ran `make test_porting` on Linux before making commit 6319bad96eba493c5f3bc7857e1c58687c461d70, and I re-tested on FreeBSD after making that commit.

The new entry in AUTHORS shows up fine in `vim`:
```
Håkon Hægland                  <hakon.hagland@gmail.com>
```

Thank you very much.
Jim Keenan

**Comment by tonycoz** [comment]
The linked URL has expired, does this happen in a particular branch? Or on a particular platform?

**Comment by jkeenan** [comment]
> The linked URL has expired, does this happen in a particular branch? Or on a particular platform?

Well, that's another indication that the usefulness of these github.actions is circumscribed. :-(

The branch was blead.  The platform was probably Linux.

**Comment by xenu** [comment]
log urls aren't shareable, linking to the run's information page works better, e.g.: https://github.com/Perl/perl5/actions/runs/104572207

**Comment by jkeenan** [comment]
Okay, I've figured out what happened.

First, prompted by @xenu's comment I figured out where on github one can review the results of any particular github.action.  Going to https://github.com/Perl/perl5/actions enables one to page through the particular actions in reverse chronological order.  When I got to the [5th page](https://github.com/Perl/perl5/actions?page=5) in that series, I realized that the entry I reported in the OP was actually a test failure in the commit author's **original pull request**.  And, of course, most pull requests from first-time authors will inevitably report a failure in `t/porting/authors.t` -- because they're not yet listed in `AUTHORS`!

I merged a patch from the submitter's p.r. locally, added his name to `AUTHORS`, squashed the two commits, and then re-ran `make test_porting` prior to pushing to blead.  **That** commit did get PASS from github.actions, as reported on the next most recent page at the URL listed above.  So I was mistaking a github.action report on a p.r. for my final commit to blead.

Closing this issue.

Thank you very much.
Jim Keenan
